koku/subs/subs_data_extractor.py

```diff
@@ -22,11 +22,89 @@
 from reporting.models import SubsIDMap
 from reporting.models import SubsLastProcessed
 from reporting.provider.aws.models import TRINO_LINE_ITEM_TABLE as AWS_TABLE
+from reporting.provider.azure.models import TRINO_LINE_ITEM_TABLE as AZURE_TABLE
+
 
 LOG = logging.getLogger(__name__)
 
 TABLE_MAP = {
     Provider.PROVIDER_AWS: AWS_TABLE,
+    Provider.PROVIDER_AZURE: AZURE_TABLE,
+}
+
+ID_COLUMN_MAP = {
+    Provider.PROVIDER_AWS: "lineitem_usageaccountid",
+    Provider.PROVIDER_AZURE: "COALESCE(subscriptionid, subscriptionguid)",
```

Comment:
Little side note on these, is there a chance we will need to add some NULLIF's to these after the trino changes I'm making. Coalesce will grab the first string it finds, so since the trino schema stuff sets empty columns to "" we might grab the wrong values now. See: https://github.com/project-koku/koku/pull/4817/files#diff-bbae4b8cefdc93f92046d1cd5a894845f01613fd7f4d1b9d0d2c19e7db198d3aR24

```suggestion
    Provider.PROVIDER_AZURE: "COALESCE(NULLIF(subscriptionid, ""), subscriptionguid)",
```